src/org/zaproxy/zap/authentication/AuthenticationHelper.java

```diff
@@ -31,8 +34,16 @@ public AuthenticationHelper(HttpSender httpSender, SessionManagementMethod sessi
 	private static final Logger log = Logger.getLogger(AuthenticationHelper.class);
 
 	private static final String HISTORY_TAG_AUTHENTICATION = "Authentication";
+	public static final String AUTH_SUCCESS_STATS = "stats.auth.success";
+	public static final String AUTH_FAILURE_STATS = "stats.auth.failure";
 
-	public static void notifyOutputAuthSuccessful() {
+	public static void notifyOutputAuthSuccessful(HttpMessage msg) {
```

Comment:
This breaks binary compatibility, since the next main release is expected to be binary compatible the "old method" should be deprecated instead.